samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c

```diff
@@ -71,31 +71,21 @@ BT_MESH_MODEL_PUB_DEFINE(gen_level_cli_pub_s0, NULL, 2 + 7);
 /* Definitions of models publication context (End) */
 
 /* Definitions of models user data (Start) */
-struct generic_onoff_state gen_onoff_srv_root_user_data = {
-	.transition = &lightness_transition,
-};
+struct generic_onoff_state gen_onoff_srv_root_user_data;
```

Comment:
Sinne these are global variables (non-static), the above change means that it is not anymore initialized here. Where are you then initializing it?

Reply:
No need to initialize them here.

But some variables under those structures are initialized under **light_default_var_init()**
defined in **main.c** file.

They will re-initialized under light_default_status_init() which will get called after reading
variable stored under "/ps" settings branch.

Reply:
Other variable will be initialized as zero. And they will get automatically initialized
further once the state_binding() function get called in light_default_status_init(void).

Kindly refer main.c for details.

Reply:
Done !!